.github/workflows/rebase.yml

```
@@ -0,0 +1,21 @@
+name: Rebase
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  rebase:
+    name: Automatic rebase
+    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/rebase')
```

Can we change this to a full equality match lest we accidentally rebase it with other comments?
e.g. `just do a /rebase`.

```suggestion
    if: github.event.issue.pull_request != '' && github.event.comment.body == '/rebase'
```